api/src/main/kotlin/eu/codeloop/thehub/jobs/JobQueryController.kt

```diff
@@ -15,7 +18,11 @@ class JobQueryController(private val jobQueryService: JobQueryService, private v
         @RequestParam keyword: String?,
         @RequestParam q: String?,
         @RequestParam hasStar: Boolean?,
-        pageable: Pageable
+        @PageableDefault
+        @SortDefault.SortDefaults(
+            SortDefault(sort = ["approvedAt"], direction = Sort.Direction.DESC),
+            SortDefault(sort = ["title"], direction = Sort.Direction.ASC)
```

Comment:
Import static so it would look like that:

```kotlin
@SortDefaults(
            SortDefault(sort = ["approvedAt"], direction = DESC),
            SortDefault(sort = ["title"], direction = ASC)
)
```